api/src/domain/user/maskTaxId.ts

```diff
@@ -0,0 +1,5 @@
+export const maskTaxId = (taxId: string): string => {
+  return taxId.length == 12
+    ? `*******${taxId.slice(7, taxId.length)}`
+    : `*****${taxId.slice(5, taxId.length)}`;
```

Comment:
I believe the whole reason that we talked about extracting `*` into a shared variable was so that anytime it was referenced in code (the tests are fine) that we should be referencing it by variable name